### PR TITLE
Cleaner Config File

### PR DESCRIPTION
**Pull request by sithlord48**
- Starting to Make the config Cleaner and Easier to read.

### CODE REVIEW

File: misc/FFNx.cfg [hunk 1]
```diff
@@ -15,46 +15,73 @@
...
+# Default (value = 0):
+# - Window mode will use 640x480
+# - Fullscreen mode will use your desktop resolution
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~
 #window_size_x = 0
 #window_size_y = 0
 
+#[INTERNAL RESOLUTION SCALE]
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Internal Scale used it Perserve Aspect is not enabled.
```
Comment (julianxhokaxhiu) [inline misc/FFNx.cfg:50]: Needs a rephrase or something is missing?

File: misc/FFNx.cfg [hunk 2]
```diff
@@ -15,46 +15,73 @@
...
+# By default is set to four times the calculated game rendering resolution, that is nearest to 640x480
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~
 #internal_resolution_scale = 4
 
-# Preserve original game aspect ratio (4:3) by adding black bars on the left and right side.
-# If you wish to have it fully stretched, keep this option off. Be aware the game may look wrong though.
+#[PRESERVE ASPECT]
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Preserve original game aspect ratio of (4:3) by adding black bars on the left and right side (if needed)
+# When off the game will be stretched to fit the windows aspect ratio; Be aware the game may look wrong though.
```
Comment (julianxhokaxhiu) [inline misc/FFNx.cfg:59]: windows -> window

File: misc/FFNx.cfg [hunk 3]
```diff
@@ -63,59 +90,136 @@
 # - 8: 8x
 # - 16: 16x
 # If you set any value that is different than those, it will pick the higher option ( eg. if you pick 6 it will be 8, if you pick 1 it will be 2, and so on ).
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~
 #enable_antialiasing = 0
 
+#[ANISOTROPIC]
```
Comment (julianxhokaxhiu) [inline misc/FFNx.cfg:96]: Missing `~~~~~~~~~~~~~~~~~~~~~~~~~~~` sections?

File: misc/FFNx.cfg [hunk 4]
```diff
@@ -15,46 +15,73 @@
...
+#[RENDERING BACKEND]
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~
 # Available choices are:
-### SUGGESTED:
 # - OpenGL ( default )
 # - Direct3D11 ( valid alternative for OpenGL)
-### NOT SUGGESTED (TECHNICAL POC):
-# - Vulkan ( slow performance)
-# - Direct3D12 ( slow performance )
+# - Vulkan ( Tech Preview: Not Recommened )
```
Comment (myst6re) [inline misc/FFNx.cfg:27]: Recommened?

File: misc/FFNx.cfg (hunk 1 above)
Comment (sithlord48) [reply misc/FFNx.cfg:50]: Can you describe what this setting is doing for me ?

File: misc/FFNx.cfg (hunk 2 above)
Comment (julianxhokaxhiu) [reply misc/FFNx.cfg:60]: There's still `windows` instead of `window` :P

File: misc/FFNx.cfg [hunk 5]
```diff
@@ -63,59 +91,138 @@
...
+# The type of file that the ffmpeg layer will search for. Default is avi.
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#ffmpeg_video_ext = avi
 
-# STEAM ONLY!
+
+###############################
+# MUST SET FOR VERSIONS BELOW
+# FF7 2012   FF7 STEAM
+# FF8 2013   FF8 STEAM
```
Comment (julianxhokaxhiu) [inline misc/FFNx.cfg:172]: FF8 2013 and FF8 Steam isn't the game edition? Or am I missing something?

File: misc/FFNx.cfg (hunk 2 above)
Comment (sithlord48) [reply misc/FFNx.cfg:60]: should be window's .. 
Must have missed this in my update

File: misc/FFNx.cfg (hunk 5 above)
Comment (sithlord48) [reply misc/FFNx.cfg:172]: No the 2013 release is done by square enix  before a steam release but idk that they still sell it . much like the 2012 release of FF7 it seams to have bee discontinued in place of the Steam realease

Comment (julianxhokaxhiu) [reply misc/FFNx.cfg:172]: Good to know :) So there's an eStore release of FF8 then...guess I need to update the README too. Thanks!

**Comment by julianxhokaxhiu** [comment]
Great work :) Thank you!

**Pull request merged**